pinot-spi/src/main/java/org/apache/pinot/spi/utils/RebalanceConfigConstants.java

```diff
@@ -19,18 +19,13 @@
 package org.apache.pinot.spi.utils;
 
 /**
- * Constants for rebalance config properties
+ * Defaults for rebalance configs are defined here. But use RebalanceConfig to define the configs, and deprecate
```

Comment:
Should we just remove this class? (since we already moved 2 fields)

Reply:
love to. no more references to the config names defined here after this cleanup, but there are still references to the default config values defined here.